app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectPresenter.kt

```diff
@@ -92,6 +97,10 @@ class DeviceConnectPresenter(deviceActivity: DeviceActivity, manager: WifiManage
         checkPermissions = b
     }
 
+    override fun checkWifiEnabled() {
+        isWifiEnabled = mWifiManager.isWifiEnabled
```

Comment:
Create an issue to remove hungarian notation

Reply:
Created https://github.com/fossasia/susi_android/issues/1689